content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst

```diff
@@ -0,0 +1,121 @@
+with the purchase order. Clicking on the :guilabel:`Receipt` smart button will show the receipt
+order.
+
+.. image:: reciepts_three_steps/three-step-purchase-receipt.png
+   :align: center
+   :alt: After confirming a purchase order, a Receipt smart button will appear.
+
+Process a receipt
+-----------------
+
+One receipt and two internal transfers (one transfer to quality, and a subsequent transfer to stock)
+will be created once the purchase order is confirmed. To view these transfers, go to
+:menuselection:`Inventory --> Operations --> Transfers`.
+
+.. image:: reciepts_three_steps/three-step-transfers.png
+   :align: center
+   :alt: The status of the three receipt transfers will show which operation is ready and which ones
+         are waiting another operation.
+
+The status of the receipt transferring the product to the input location will be :guilabel:`Ready`,
+since the receipt must be processed before any other operation can occur. The status of the two
+internal transfers will be :guilabel:`Waiting Another Operation`, since the transfers cannot be
+processed until the linked step before each transfer is completed.
+
+The status of the first internal transfer to quality will only change to :guilabel:`Ready` when the
+receipt has been marked :guilabel:`Done`. The status for the second internal transfer to stock will
+be marked :guilabel:`Ready` only after the transfer to quality has been marked :guilabel:`Done`.
+
+The receipt can also be found in the :guilabel:`Inventory` application. In the overview dashboard,
+click the :guilabel:`1 To Process` smart button in the :guilabel:`Receipts` Kanban card.
+
+.. image:: reciepts_three_steps/three-step-receive-kanban.png
+   :align: center
+   :alt: One Receipt ready to process in the Inventory Overview Kanban view.
+
+Click on the receipt associated with the purchase order, then click :guilabel:`Validate` to complete
+the receipt and move the product to the :guilabel:`Quality Location`.
+
+.. image:: reciepts_three_steps/validate-three-step-receipt.png
+   :align: center
+   :alt: Validate the receipt by clicking Validate, and the product will be transferred to the
+         WH/Quality location.
+
+Process a transfer to Quality Control
+-------------------------------------
+
+Once the product is in the :guilabel:`Input Location`, the internal transfer is ready to move the
+product to :guilabel:`Quality Control`. In the inventory overview dashboard, click the :guilabel:`1
+To Process` smart button in the :guilabel:`Internal Transfers` Kanban card.
+
+.. image:: reciepts_three_steps/three-step-quality-transfer.png
+   :align: center
+   :alt: One Internal Transfer ready to process in the Inventory Overview Kanban view.
+
+Click on the transfer associated with the purchase order, then click :guilabel:`Validate` to
+complete the transfer and move the product to the :guilabel:`Quality Control` location. Once the
+transfer is validated, the product is ready for the quality inspection.
```

Comment:
ready for the quality inspection, but is not available for manufacturing or delivery orders.